docs/semgrep-ci/network-broker.md

```diff
@@ -0,0 +1,110 @@
+---
+slug: network-broker
+title: Network broker
+hide_title: false
+description: Learn how to set up the Semgrep Network Broker, which facilitates secure access between Semgrep and your private network.
+tags:
+  - Semgrep in CI
+  - Network broker
+---
+
+# Set up the Semgrep network broker
+
+The Semgrep network broker facilitates secure access between Semgrep and your private network. It accomplishes this by establishing a Wireguard VPN tunnel with the Semgrep infrastructure, then proxying **inbound** (from Semgrep to your network) HTTP requests through this tunnel. This approach allows Semgrep to interact with on-premise resources without having to expose them to the public internet.
+
+Examples of inbound traffic include:
+
+- Pull or merge request (PR or MR) comments
+- JIRA integrations
```

Comment:
I'm not sure Jira integration is inbound, is it? Don't we only create issues for now?

Reply:
I think it's inbound to Jira for on-prem deployments?

Reply:
We don't support on-prem Jira right now (although it is something we're looking at).
This list is from the original NB docs (I was reading them yesterday) so maybe @tpetr can comment on that while he's here :)